{"uuid":"92aadae9-fd10-49c9-ab1c-8e9680d516ab","historyId":"8909df0c5508604a9087476fdf99c6f1","fullName":"com.manorama.testClasses.SmokePath.search_functions","labels":[{"name":"package","value":"com.manorama.testClasses.SmokePath"},{"name":"testClass","value":"com.manorama.testClasses.SmokePath"},{"name":"testMethod","value":"search_functions"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.manorama.testClasses.SmokePath"},{"name":"host","value":"DESKTOP-EI1707P"},{"name":"thread","value":"18248@DESKTOP-EI1707P.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"search functions","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tException occured in while validating Failed while clicking on element:click_on_department.Exceptionelement click intercepted: Element <button class=\"btn btn-flat btn-xs\" id=\"btnSelectDept\" type=\"button\" title=\"\" onclick=\"GetDepartmentList(event);\">...</button> is not clickable at point (305, 373). Other element would receive the click: <div id=\"divMsgBack\" style=\"background: rgba(0, 0, 0, 0.7); width: 100%; height: 100%; top: 0px; left: 0px; z-index: 2000; position: fixed;\">...</div>\n  (Session info: chrome=131.0.6778.140)\nBuild info: version: '4.17.0', revision: 'e52b1be057*'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [5851a3bf522b29a8869e6d7c10e778ca, clickElement {id=f.EDB473FD8A1AE48F6DC868E78736FCAF.d.D560D2585BFF162BEE450F95C226A4F5.e.409}]\nCapabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.140, chrome: {chromedriverVersion: 131.0.6778.108 (3b014839fbc..., userDataDir: C:\\Users\\Admin\\AppData\\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50050}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50050/devtoo..., se:cdpVersion: 131.0.6778.140, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}\nElement: [[ChromeDriver: chrome on windows (5851a3bf522b29a8869e6d7c10e778ca)] -> xpath: //button[@id='btnSelectDept' and @onclick='GetDepartmentList(event);']]\nSession ID: 5851a3bf522b29a8869e6d7c10e778ca expected [true] but found [false]","trace":"java.lang.AssertionError: The following asserts failed:\n\tException occured in while validating Failed while clicking on element:click_on_department.Exceptionelement click intercepted: Element <button class=\"btn btn-flat btn-xs\" id=\"btnSelectDept\" type=\"button\" title=\"\" onclick=\"GetDepartmentList(event);\">...</button> is not clickable at point (305, 373). Other element would receive the click: <div id=\"divMsgBack\" style=\"background: rgba(0, 0, 0, 0.7); width: 100%; height: 100%; top: 0px; left: 0px; z-index: 2000; position: fixed;\">...</div>\n  (Session info: chrome=131.0.6778.140)\nBuild info: version: '4.17.0', revision: 'e52b1be057*'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [5851a3bf522b29a8869e6d7c10e778ca, clickElement {id=f.EDB473FD8A1AE48F6DC868E78736FCAF.d.D560D2585BFF162BEE450F95C226A4F5.e.409}]\nCapabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.140, chrome: {chromedriverVersion: 131.0.6778.108 (3b014839fbc..., userDataDir: C:\\Users\\Admin\\AppData\\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50050}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50050/devtoo..., se:cdpVersion: 131.0.6778.140, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}\nElement: [[ChromeDriver: chrome on windows (5851a3bf522b29a8869e6d7c10e778ca)] -> xpath: //button[@id='btnSelectDept' and @onclick='GetDepartmentList(event);']]\nSession ID: 5851a3bf522b29a8869e6d7c10e778ca expected [true] but found [false]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)\r\n\tat com.manorama.framework.BaseTest.testResult(BaseTest.java:132)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)\r\n\tat org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)\r\n\tat org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1734523260422,"stop":1734523260423}